Update README (#54)
AICoE/meteor: README.md

# Project Meteor

> Project Meteor is an AICoE team effort to build an end-end Data Science product showcasing the usefulness of Thoth and AI Ops capabilities along with the integration of  Open Data Hub/Red Hat OpenShift Data Science.

Reviewer: please rephrase, dont think about what we want to achieve with it, think about what we do for... whom? ...

Author: updated the description

> It is an AICoE combined team effort to provide a product for Data Scientists and other users alike where they can interact and leverage our services, tools and technology for developing various applications. The product can also help in introducing, sharing and promoting our team's capabilities with others.

Reviewer 2: are you able to give some context on AICoE for non-AICoE-ers? :)

Author: yes good idea, ill include a sentence for this

Reviewer 3: ```suggestion
Project Meteor is a combined effort across the AI CoE to provide a single tool for data scientists and other users where they can interact with, explore and leverage all of our services, tools and technologies for developing intelligent applications. 
```

> ## Product

Reviewer 2: Do we want to add anything about how this project is supported by the Operate First environment? Or better to keep it agnostic?

Author: I think we can mention that, ill add it in

Reviewer 3: is "product" the right word? Doesn't Red Hat have a pretty serious distinction between "projects" and "products" maybe we call it a tool?

The product is an interactive [Elyra tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) web application that is rendered as a [JupyterBook](https://jupyterbook.org/) environment by taking as an input the user's GitHub repository (user's fork of the Elyra tutorial repository) and runs the build/deployment pipelines in the backend to render the notebook environment as the final output. You can also take a look at the product flowchart for more details [here](https://github.com/AICoE/meteor/blob/main/docs/project-flowchart.md).

### Elyra Tutorial

The Elyra tutorial is used to discuss the interface between Data Science and Dev/DevOps using project templates, pipelines and bots. Moreover, it wants to highlight that Data Scientists are not so different from developers and DevSecOps practices and tools can be applied to MLOps ones.

The demo application used in this tutorial is the "hello world" for AI: MNIST Classification. To learn more about Elyra, you can check out their [repository](https://github.com/elyra-ai/elyra).

> 

Reviewer: maybe its worth mentioning that the ML app itself is not in focus, its the workflow, the service, the product that we want to show off.

# Project Meteor

Similar to how a meteoroid entering the atmosphere, becomes visible as a meteor and finally hitting the Earth's surface as a meteorite, we aim to create better visibility and impact of the AICoE team's work through project *Meteor*.

AICoE is the Artificial Intelligence Center of Excellence team, part of Red Hat's Office of the CTO established to enable Analytics, Machine Learning, and AI cross-functionally and bundle forward-looking R&D. The team's focus is to apply AI to our core business and services through an Open Source driven development to enable user's needs and foster open collaboration.

It is an AICoE combined team effort to provide a product for Data Scientists and other users alike where they can interact and leverage our services, tools and technology for developing various applications. The product can also help in introducing, sharing and promoting our team's capabilities with others.

## Product

> The product is an interactive [Elyra tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) web application that is rendered as a [JupyterBook](https://jupyterbook.org/) environment by taking as an input the user's GitHub repository (user's fork of the Elyra tutorial repository) and runs the build/deployment pipelines in the backend to render the notebook environment as the final output. You can also take a look at the product flowchart for more details [here](https://github.com/AICoE/meteor/blob/main/docs/project-flowchart.md).

Reviewer 3: ```suggestion
The product is an interactive [AiDevSecOps tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) and web application that is rendered as both a static [JupyterBook](https://jupyterbook.org/) and an interactive Jupyter Lab environment by taking as an input the user's GitHub repository (specifically a fork of the tutorial repository) and runs the build/deployment pipelines in the background to render the notebook environments as the final output.
```
Can you just show the flow chart as an image here instead of linking to it?

The product is supported by and hosted on the [Operate First](https://www.operate-first.cloud/) environment. Operate First is an initiative started at the Red Hat's Office of the CTO, to work towards making operations as fundamental as functionality in the upstreams they contribute to and the products they produce. To learn more about Operate First and how you can be involved, you can visit our [website](https://www.operate-first.cloud/) or our [GitHub community](https://github.com/operate-first)!

> ### Elyra Tutorial

Reviewer 3: ```suggestion
### AiDevSecOps Tutorial
```

> The Elyra tutorial is used to discuss the interface between Data Science and Dev/DevOps using project templates, pipelines and bots. Moreover, it wants to highlight that Data Scientists are not so different from developers and DevSecOps practices and tools can be applied to MLOps ones.

Reviewer 3: ```suggestion
The tutorial is used to discuss the interface between data science and Dev/DevOps using project templates, pipelines and bots. Moreover, it highlight that data scientists are not so different from developers and DevSecOps practices and tools can (and should) be applied to MLOps.
```

> The demo application used in this tutorial is the "hello world" for AI: MNIST Classification. The ML application developed itself is not the main focus of the product, but its the various services, workflows, and pipeline integrations supported by the product that we would like to highlight and provide for our users. To learn more about Elyra, you can check out their [repository](https://github.com/elyra-ai/elyra).

Reviewer 3: ```suggestion
The demo application used in this tutorial is the "hello world" for AI: MNIST Classification. The ML application developed itself is not the main focus of the product, but its the various services, workflows, and pipeline integrations supported by the product that we would like to highlight and provide for our users.
```

## Project Workstreams

> In order to support the goals of the project, we have 3 workstreams defined to effeciently manage and focus on the different aspects of the project. The workstreams are as follows:

Reviewer 3: ```suggestion
In order to support the goals of the Project Meteor, we have crated 3 workstreams defined to efficiently manage and focus on the different aspects of the project.
```

### Team Comet

Team Comet focuses on the technical planning and implementation of the product. Their main responsibilities include designing the technical roadmap/design of the product, implementing the backend CI/build pipelines for deploying the web application and maintaining the application.

### Team Shooting Star

> Team Shooting Star focuses on polishing the Elyra tutorial, as well as other AICoE tutorial and workshops to be well adopted into the product. Team members will work on improving the interactive tutorial both in terms of the content and the user experience from a Data Scientist perspective.

Reviewer 3: ```suggestion
Team Shooting Star focuses on polishing the tutorial and project content, as well as other AICoE tutorials and workshops to be well adopted into the product. Team members will work on improving the interactive tutorials both in terms of the content and the user experience from a data scientist perspective.
```
